### ISSUE TITLE
Catch CodeCommit rate limit errors

### ISSUE THREAD

**Issue by symroe** [DemocracyClub/LGSF#73]
A lot of scrapers are returning:

`An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded`

I suspect this is because on ModGov scrapers we're calling commit on each row in the data as quickly as we can, meaning we're creating a commit per councillor very quickly.

We should either wait between calls to commit, or find a way to batch all the commits up in to a single commit somehow.

**Comment by symroe** [comment]
No longer needed after #254
